packaging/nuget/nservicebus.ravendb.nuspec

```diff
@@ -16,7 +16,7 @@
     <tags>nservicebus servicebus msmq cqrs publish subscribe ravendb</tags>
     <dependencies>
       <dependency id="NServiceBus" version="[5.2.8, 6.0.0)" />
-      <dependency id="RavenDB.Client" version="[3.0.3800, 4.0.0)" />
+      <dependency id="RavenDB.Client" version="[3.0.3800, 3.1.0)" />
```

Comment:
Hi,

I just wanted to check if this was still case? That users on RavenDB 3.5.2 and NServiceBus 5.x cannot use NServiceBus.RavenDB 3.x? 

Is the ONLY solution to upgrade to NServicebus 6? We just spent 3 months upgrading our solution to Raven 3.5.2 only to find that our sagas dont work with NServiceBus 5.x